pilot/pkg/model/telemetry.go

```diff
@@ -1052,3 +1053,32 @@ func generateStatsConfig(class networking.ListenerClass, metricsCfg telemetryFil
 func disableHostHeaderFallback(class networking.ListenerClass) bool {
 	return class == networking.ListenerClassSidecarInbound || class == networking.ListenerClassGateway
 }
+
+func resourceAttributes(proxy *Proxy) *otlpcommon.KeyValueList {
+	// TODO(zirain): generate resource attributes for VM
+	if proxy.IsVM() {
+		return nil
+	}
+
+	return &otlpcommon.KeyValueList{
+		Values: []*otlpcommon.KeyValue{
+			{
+				Key:   "k8s.cluster.name",
+				Value: &otlpcommon.AnyValue{Value: &otlpcommon.AnyValue_StringValue{StringValue: proxy.Metadata.ClusterID.String()}},
+			},
+			{
+				Key: "k8s.service.name",
```

Comment:
are these keys defined somewhere

Reply:
come from here https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/k8s/

Reply:
I don't see k8s.service.name there, and this isn't actually the "service" anyways so it seems a bit misleading?

Reply:
my fault, correct it now.